app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/checkout/_progress-bar.less

```diff
@@ -57,8 +60,6 @@
 //  Desktop
 //  _____________________________________________
 
```

Comment:
any reason of remove this media mixin ?

Reply:
As this issue is working fine in blank theme. So I compared the code and there as well we don't have any mixin and using the progress bar code irrespective of view. So the progress bar will show in each device.